winui/Rating/ToolTip.md

```diff
@@ -1,12 +1,61 @@
 ---
 layout: post
 title: ToolTip with WinUI Rating control | Syncfusion
-description: Learn here all about Enabletooltip and tooltip format support in the rating items in WinUI rating control.
+description: Learn here all about how to enable the tooltip and format the tooltip content in WinUI Syncfusion Rating (SfRating) control.
 platform: WinUI
 control: SfRating
 documentation: ug
 ---
 
 # ToolTip in WinUI Rating
 
-This section explains the tooltip features available in the WinUI Rating control.
+The Rating control tooltip can be enabled by using the `EnableToolTip` property. It will show the rating value content as a tooltip when the mouse hovers on the control. The default value is `false`.
```

Comment:
The Rating control tooltip can be enabled by using the `EnableToolTip` property. When the mouse hovers over the control, it will show the rating value content as a tooltip. The default value is `false`.

Reply:
Changed the description content as mentioned above.